Disabling default features fails to compile

Disabling default features fails to compile the crate, e.g.
```
cargo new --bin test-bevy-rand
cd test-bevy-rand
cargo add bevy_rand --no-default-features
cargo build
```
fails with
```
   Compiling bevy_rand v0.8.0
error[E0425]: cannot find value `seed` in this scope
   --> /home/marius/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_rand-0.8.0/src/traits.rs:247:23
    |
247 |             getrandom(seed.as_mut()).expect("Unable to source entropy for seeding");
    |                       ^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `bevy_rand` (lib) due to 1 previous error
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -46,7 +46,10 @@ jobs:
       - name: Run clippy
         run: cargo clippy --workspace --all-features --all-targets -- -Dwarnings
         if: startsWith(matrix.rust, 'nightly')
-      - run: cargo test --all-features
+      - name: No Default Feature checks
+        run: cargo check --no-default-features
+      - name: Test with all features enabled
+        run: cargo test --all-features
       - name: Test wasm
         run: wasm-pack test --headless --chrome --firefox -- --all-features
         if: startsWith(matrix.os, 'ubuntu')

--- a/src/seed.rs
+++ b/src/seed.rs
@@ -73,11 +73,11 @@ where
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-
     #[cfg(feature = "serialize")]
     #[test]
     fn reflection_serialization_round_trip_works() {
+        use super::*;
+
         use bevy_prng::WyRand;
         use bevy_reflect::{
             serde::{TypedReflectDeserializer, TypedReflectSerializer},

--- a/src/traits.rs
+++ b/src/traits.rs
@@ -244,7 +244,7 @@ where
         {
             use getrandom::getrandom;
 
-            getrandom(seed.as_mut()).expect("Unable to source entropy for seeding");
+            getrandom(dest.as_mut()).expect("Unable to source entropy for seeding");
         }
 
         Self::from_seed(dest)

